snmp_cisco/metadata.csv

```diff
@@ -0,0 +1,3 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+snmp.cefcFRUPowerAdminStatus,gauge,,,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] Administratively desired FRU power state.,0,snmp,
```

Comment:
(not blocking) I'm not sure there is a need to keep `metadata.csv` if not used in `snmp_cisco/manifest.json`

Reply:
I think, one solution is to update the `ddev validate manifest`, to skip metadata.csv validation for this integration.